configure.ac

```diff
@@ -371,6 +372,7 @@ OPTION_DEFAULT_DISABLE([test_sampler], [ENABLE_TEST_SAMPLER])
 OPTION_DEFAULT_DISABLE([list_sampler], [ENABLE_LIST_SAMPLER])
 OPTION_DEFAULT_DISABLE([record_sampler], [ENABLE_RECORD_SAMPLER])
 dnl test_sampler will also build with --enable-ldms-test
+OPTION_DEFAULT_ENABLE([json_stream_sampler], [ENABLE_JSON_STREAM_SAMPLER])
```

Comment:
I don't think we have json_stream_sampler in 4.4.x

Reply:
Looks like residual ... we will remove this.